### PR TITLE
Update decimal.js

### PR DESCRIPTION
**Pull request by harryadel**
Might be nice to add it along #11867

### CODE REVIEW

File: packages/non-core/mongo-decimal/package.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ Package.describe({
 });
 
 Npm.depends({
-  "decimal.js": "10.2.1"
+  "decimal.js": "10.3.1"
```
Comment (denihs) [inline packages/non-core/mongo-decimal/package.js:7]: Hi @harryadel, sorry, but I didn't understand the purpose of bumping the version of this package without making any actual change to it. Could you explain it to me?

**Review by denihs — CHANGES_REQUESTED** [review]

**Comment by harryadel** [comment]
:thinking:

**Comment by filipenevola** [comment]
@harryadel what are the improvements and changes in this minor version update?

We will be more strict with PRs to follow our guideline for contribution. https://github.com/meteor/meteor/blob/devel/CONTRIBUTING.md#proposing-your-change

**Comment by harryadel** [comment]
@filipenevola I guess my argument would be the same as one [here](https://github.com/meteor/meteor/pull/11837#issuecomment-1033214636)

**Review by denihs — APPROVED** [review]

**Pull request merged**